ui/api.js

```diff
@@ -49,11 +49,11 @@ async function remove(request) {
 async function removeJobs(request) {
   const { status } = request.params;
 
-  if (!/(^active$)|(^processed$)|(^failed$)/.test(status)) {
-    return ok(false);
+  if (!['active', 'processed'].includes(status)) {
```

Comment:
We could also authorize deletion on `enqueued` tasks.

Reply:
We could, but it is currently not used, and would be a bit more complicated to implement (due to the part that lists the jobs)

Reply:
yep sure 👍